File: magic/src/main/java/br/com/bloder/magic/Magic.java
package br.com.bloder.magic;

import android.view.View;

import br.com.bloder.magic.internal.MagicAnimation;

/**
 * Created by bloder on 28/07/16.
 */
public class Magic {

  public void doWith(View button) {
    MagicAnimation animator = new MagicAnimation(button);
    animator.setDuration(200);
    button.startAnimation(animator);
  }
}

File: magic/src/main/java/br/com/bloder/magic/Magic.kt
package br.com.bloder.magic

import android.view.View
import br.com.bloder.magic.internal.MagicAnimation

/**
 * Created by bloder on 20/09/16.
 */
fun doWith(view : View) {
    var animation : MagicAnimation
    animation = MagicAnimation(view)
    animation.duration = 200
    view.startAnimation(animation)
}